app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -88,6 +95,10 @@ def handle_bad_request_error(err, enrollment)
       def handle_standard_error(err, enrollment)
         raise Exception::RequestEnrollException.new(err.message, err, enrollment.id)
       end
+
+      def analytics(user: AnonymousUser.new)
+        Analytics.new(user: user, request: nil, session: {}, sp: nil)
```

Comment:
Will this always have an anonymous user? I think it could be nice if we could pass in the analytics instance from the controller to the service class, which would also retain information from the request, session, and service provider. It might require some restructuring of the service class to have it be constructed as an instance, rather than using static methods.

Something like...

```rb
UspsInPersonProofing::EnrollmentHelper.new(analytics: analytics).schedule_in_person_enrollment(user, pii)
```

Reply:
addressed in 2b7f5dfc6c7748e7a2612069ce8df3c61653ae7e to include the user for this specific case